file-formats/dobj/examples/z_aff_example_dobj.dobj.json

```diff
@@ -0,0 +1,18 @@
+{

```

Comment:
Can you check whether the example file is marked with line endings CRLF? It should be LF. I have changed it twice, but it seems you replace it with every update, because your file is still marked with CRLF. ;)